.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: go
+sudo: required
```

Comment:
This tag is deprecated by travis ci, it does nothing anymore

Reply:
Done